packages/labs/analyzer/src/lib/model.ts

```diff
@@ -585,11 +602,42 @@ export class CustomElementDeclaration extends ClassDeclaration {
   constructor(init: CustomElementDeclarationInit) {
     super(init);
     this.tagname = init.tagname;
+    this.attributes = init.attributes;
     this.events = init.events;
     this.slots = init.slots;
     this.cssProperties = init.cssProperties;
     this.cssParts = init.cssParts;
   }
+
+  #addFieldToAttributeMap(
+    field: ClassField,
+    attributes: Map<string, Attribute>
+  ) {
+    if (field instanceof CustomElementField && field.attribute) {
```

Comment:
I know that `attribute: ''` is invalid, but do we want to represent it here for linters to be able to warn on? `&& field.attribute` will prevent that from going through.

Reply:
i think you're right and we should let it through. not sure how else we'd get at the original value without some hackery